CMakeLists.txt

```diff
@@ -240,6 +240,18 @@ if (CMAKE_VERSION VERSION_GREATER 3.4)
   set(ENABLE_SANITIZER "none" CACHE STRING "Add clang sanitizer to the build")
 endif ()
 
+if (NOT WIN32 AND NOT APPLE AND NOT QT_ANDROID)
+    if (EXISTS "/usr/lib64")
```

Comment:
Should this check also if we are compiling for a 64-bit target?

Reply:
I don't think so. /lib64 is only used on some linux distros, and no-one makes a 32-bit build on these. Or....